projects/bird-atlas/src/app/+grid/grid-index/grid-index-table/grid-index-table.component.ts

```diff
@@ -0,0 +1,85 @@
+import { ChangeDetectionStrategy, Component, EventEmitter, Input, OnDestroy, OnInit, Output } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { TranslateService } from '@ngx-translate/core';
+import { TableColumn } from '@swimlane/ngx-datatable';
+import { datatableClasses } from 'projects/bird-atlas/src/styles/datatable-classes';
+import { BehaviorSubject, Subject } from 'rxjs';
+import { debounceTime, filter, takeUntil } from 'rxjs/operators';
+import { AtlasGrid, AtlasGridSquare } from '../../../core/atlas-api.service';
+
+type DatatableRow = any;
+
+@Component({
+  selector: 'ba-grid-index-table',
+  templateUrl: './grid-index-table.component.html',
+  styleUrls: ['./grid-index-table.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class GridIndexTableComponent implements OnInit, OnDestroy {
+  private unsubscribe$ = new Subject<void>();
+
+  @Input() set atlasGrid(atlasGrid: AtlasGrid) {
+    this.rows = atlasGrid;
+    this.search$.next('');
+  }
+  @Output() selectYKJ = new EventEmitter<string>();
+
+  cols: TableColumn[] = [{
+    prop: 'coordinates',
+    name: this.translate.instant('ba.grid-index.coordinates'),
+    resizeable: false,
+    sortable: true,
+    width: 75
+  }, {
+    prop: 'name',
+    name: this.translate.instant('ba.grid-index.name'),
+    resizeable: false,
+    sortable: true,
+    width: 200
+  }, {
+    prop: 'birdAssociationArea.value',
+    name: this.translate.instant('ba.grid-index.birdAssociationArea'),
+    resizeable: false,
+    sortable: true,
+    width: 350
+  }];
+  filteredRows$ = new Subject<DatatableRow[]>();
+  datatableClasses = datatableClasses;
+
+  private search$ = new BehaviorSubject<string>(undefined);
+  private rows: DatatableRow[] = [];
```

Comment:
I'm wondering why any type is used here (`type DatatableRow = any` ) when more specific type could be used (AtlasGrid)